src/luger.hrl

```diff
@@ -7,9 +7,10 @@
 -define(INFO, 6).
 -define(DEBUG, 7).
 
--record(config, { app    :: string(),
-                  host   :: string(),
-                  statsd :: boolean() }).
+-record(config, { app       :: string(),
+                  host      :: string(),
+                  statsd    :: boolean(),
+                  msglength :: integer() }).
```

Comment:
Similar, I think calling this variable `max_msg_len` is a bit more descriptive of what the variable does.